source/mapping.d

```diff
@@ -42,17 +42,17 @@ M[VK_OEM_4]         = [mCH("dead_acute", '\u02ca'),     mCH("dead_tilde", '\u02d
 M['3']              = [mVK("3", '3'),                   mCH("U2113", 'ℓ'),              mCH("threesuperior", '³'),          mCH("numerosign", '№'),             mCH("threesubscript", '₃'),    mCH("logicaland", '∧')];
-M['4']              = [mVK("4", '4'),                   mCH("guillemotright", '»'),     mCH("U203A", '›'),                  mCH("U22EE", '\u22ee'),             mCH("femalesymbol", '♀'),      mCH("uptack", '⊥')];
+M['4']              = [mVK("4", '4'),                   mCH("guillemotright", '»'),     mCH("U203A", '›'),                  VOID_KEY,                           mCH("femalesymbol", '♀'),      mCH("uptack", '⊥')];
 M['5']              = [mVK("5", '5'),                   mCH("guillemotleft", '«'),      mCH("U2039", '‹'),                  mCH("periodcentered", '·'),         mCH("malesymbol", '♂'),        mCH("U2221", '∡')];
 M['6']              = [mVK("6", '6'),                   mCH("dollar", '$'),             mCH("cent", '¢'),                   mCH("sterling", '£'),               mCH("U26A5", '⚥'),             mCH("U2225", '∥')];
 M['7']              = [mVK("7", '7'),                   mCH("EuroSign", '€'),           mCH("yen", '¥'),                    mCH("currency", '¤'),               mCH("U03F0", 'ϰ'),             mCH("rightarrow", '→')];
-M['8']              = [mVK("8", '8'),                   mCH("doublelowquotemark", '„'), mCH("singlelowquotemark", '‚'),     VOID_KEY,                           mCH("leftanglebracket", '⟨'),   mCH("infinity", '∞')];
+M['8']              = [mVK("8", '8'),                   mCH("doublelowquotemark", '„'), mCH("singlelowquotemark", '‚'),     mVK("Tab", VK_TAB),                 mCH("leftanglebracket", '⟨'),   mCH("infinity", '∞')];
 M['9']              = [mVK("9", '9'),                   mCH("leftdoublequotemark", '“'),mCH("leftsinglequotemark", '‘'),    mVK("KP_Divide", VK_DIVIDE),        mCH("rightanglebracket", '⟩'),  mCH("variation", '∝')];
 M['0']              = [mVK("0", '0'),                   mCH("rightdoublequotemark", '”'),mCH("rightsinglequotemark", '’'),  mVK("KP_Multiply", VK_MULTIPLY),    mCH("zerosubscript", '₀'),     mCH("emptyset", '∅')];
 
 // We need to map Return so that compose knows about it
 M[VK_RETURN]        = [mVK("Return", VK_RETURN),        mVK("Return", VK_RETURN),       mVK("Return", VK_RETURN),           mVK("Return", VK_RETURN),           mVK("Return", VK_RETURN),       mVK("Return", VK_RETURN)];
-M[VK_TAB]           = [mVK("Tab", VK_TAB),              mVK("Tab", VK_TAB),             mCH("Multi_key", '♫'),              mVK("Tab", VK_TAB),                 VOID_KEY,                       VOID_KEY];
+M[VK_TAB]           = [mVK("Tab", VK_TAB),              mVK("Tab", VK_TAB),             mCH("Multi_key", '♫'),              VOID_KEY,                           VOID_KEY,                       VOID_KEY];
```

Comment:
Hier sehe ich in der Abweichung von der Spec einen echten Mehrwert. Wenn Mod4-Lock aktiviert ist kann man damit nachwievor die normale Tab-Taste benutzen, z. B. beim Ausfüllen von Tabellen

Reply:
Es gibt bereits ein linkshändiges Tab auf Ebene 4: auf dem Ö. Das rechtshändige (gespiegelt von der Numlock-Taste) ist dann auf der 8. Jedoch frage ich mich genauso wie du, warum man das Tab nicht eben auf derselben Taste gelassen hat.

Da das Feld in der Referenz frei ist, halte ich die Abweichung für eine gute Idee. (Solange niemand einen Nachteil berichtet.) Ich ändere den PR entsprechend.